academy/3-my-own-chain/migration.md

```diff
@@ -148,7 +150,7 @@ To give the new v2 information a data structure, you need the following:
     }
     ```
 
-3. Add a **leaderboard rung structure** to be repeated inside the leaderboard: this stores the information of a player scoring high enough to be included in the leaderboard. It is not meant to be kept directly in storage as it is only a part of the leaderboard. Instead of involving Ignite CLI, create the structure by hand in its own file:
+3. Add a **leaderboard rung structure** to be repeated inside the leaderboard: this stores the information of a player scoring high enough to be included in the leaderboard. It is not meant to be kept directly in storage as it is only a part of the leaderboard. Instead of involving Ignite CLI, create the structure by hand in its file:
```

Comment:
By removing _own_, the reader will not know that this is a new file that they have to create. They will look for it in the folder structure and tell us on Discord that it does not exist.

```suggestion
3. Add a **leaderboard rung structure** to be repeated inside the leaderboard: this stores the information of a player scoring high enough to be included in the leaderboard. It is not meant to be kept directly in storage as it is only a part of the leaderboard. Instead of involving Ignite CLI, create the structure by hand in a new file:
```

Reply:
I would recommend reverting to "in its own file".

Reply:
Now I kinda like the "in a new file" 🤣

Reply:
haha, then let it be so